### PR TITLE
fix version command

### PR DESCRIPTION
**Pull request by fuyutsuki**
Not to mention.

### CODE REVIEW

**Comment by CookieCode** [comment]
what are you changed ?

File: src/pocketmine/command/defaults/VersionCommand.php [hunk 1]
```diff
@@ -49,11 +49,8 @@ public function execute(CommandSender $sender, $currentAlias, array $args){
 				$sender->getServer()->getName(),
 				$sender->getServer()->getPocketMineVersion(),
 				$sender->getServer()->getCodename(),
-				"\n",
-				$sender->getServer()->getApiVersion(),
-				"\n",
 				$sender->getServer()->getVersion(),
-				"\n",
+				$sender->getServer()->getApiVersion(),
 				ProtocolInfo::CURRENT_PROTOCOL
```
Comment (fuyutsuki) [inline src/pocketmine/command/defaults/VersionCommand.php]: Here

File: src/pocketmine/lang/locale/eng.ini [hunk 2]
```diff
@@ -197,12 +197,7 @@ pocketmine.server.networkError=[Network] Stopped interface {%0} due to {%1}
 pocketmine.server.networkStart=Opening server on {%0}:{%1}
 pocketmine.server.info=This server is running {%0}{%1} "{%2}" (API {%3})
 
-pocketmine.server.info.extended.title=-----Server information-----
-pocketmine.server.info.extended1=This server is running {%0}{%1} {%2}
-pocketmine.server.info.extended2=PHP version: {%0}
-pocketmine.server.info.extended3=API: {%0}
-pocketmine.server.info.extended4=Target client: Minecraft PE {%0}
-pocketmine.server.info.extended5=Protocol version: {%0}
+pocketmine.server.info.extended=This server is running {%0} {%1} 「{%2}」 implementing API version {%3} for Minecraft: PE {%4} (protocol version {%5})
```
Comment (fuyutsuki) [inline src/pocketmine/lang/locale/eng.ini]: Here

File: src/pocketmine/lang/locale/jpn.ini [hunk 3]
```diff
@@ -228,11 +228,7 @@ pocketmine.server.networkError=[Network] {%1}によって{%0}のインターフ
 pocketmine.server.networkStart={%0}:{%1}上でサーバーを開始しています
 pocketmine.server.info=このサーバーは{%0}のバージョン{%1}「{%2}」(API {%3})で動作しています
 
-pocketmine.server.info.extended.title=-----サーバーの情報-----
-pocketmine.server.info.extended1=このサーバーは {%0}{%1} を実行しています (コードネーム "{%2}")
-pocketmine.server.info.extended2=PHPバージョン: {%0}
-pocketmine.server.info.extended3=API: {%0} (iTX APIバージョン {%1})
-pocketmine.server.info.extended4=目的のクライアント: Minecraft PE {%0} (プロトコルバージョン {%1})
+pocketmine.server.info.extended=このサーバーが実行している{%0}のバージョンは{%1}「{%2}」Minecraft PE {%3}用実装APIバージョン{%4}(プロトコルバージョン{%5})です
```
Comment (fuyutsuki) [inline src/pocketmine/lang/locale/jpn.ini]: and here

**Comment by fuyutsuki** [comment]
For terracotta it is not edited. It is what git software did without permission.

**Comment by CookieCode** [comment]
Sorry but your updatedoes not help the software. Try to make more useful updates...

**Pull request closed without merging**